scheduler/drilling-wits-depth-median/src/app.py

```diff
@@ -0,0 +1,70 @@
+from typing import Dict
+
+from corva import Api, Cache, Logger, ScheduledDepthEvent, scheduled
+
+from src import api as api_lib
+from src import models, service
+from src.configuration import SETTINGS
+
+
+@scheduled
+def app(event: ScheduledDepthEvent, api: Api, cache: Cache):
+    records = api_lib.get_records(
```

Comment:
This is an awkward interface with the duplicate variable names. I know this is to add extra functionality to the API, but maybe we can rename the `src.api` module?